universal-login-sdk/lib/config/SdkConfig.ts

```diff
@@ -1,6 +1,6 @@
-import {Message, ObservedToken} from '@universal-login/commons';
+import {PartialRequired, SignedMessage, ObservedToken} from '@universal-login/commons';
 
 export interface SdkConfig {
-  paymentOptions: Message;
+  paymentOptions: PartialRequired<SignedMessage, 'gasPrice' | 'gasLimit' | 'operationType' | 'value' | 'data'>;
```

Comment:
Maybe add new type `MessageDefaults`

Reply:
Why not just partial? Why value or data would be required?

Reply:
PartialRequired clearly defines what we can put in the config and in the sdk constructor we use DeepPartial which does all fields Partial